gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -172,3 +178,8 @@ message GetRouteHintsResponse {
   // The route hints to the associated lightning node
   repeated RouteHint route_hints = 1;
 }
+
+message UpdateFedimintScidsRequest {
+  // The short channel ids of the associated lightning node
```

Comment:
nit: comment can be updated to mention that the short channel ids correspond to federations that are connected to a gateway

Reply:
updated